packages/react-storage/src/components/StorageBrowser/views/Controls/Navigate.tsx

```diff
@@ -85,47 +85,53 @@ function NavigateContainer({
   );
 }
 
-export function NavigateControl(): React.JSX.Element {
-  const [{ history, location }, handleUpdateState] = useControl('NAVIGATE');
-  const [{ isLoading }, handleUpdateList] = useAction('LIST_LOCATION_ITEMS');
-  const [, handleLocationActionsState] = useControl('LOCATION_ACTIONS');
+export function NavigateControl({
```

Comment:
For my own understanding. Is this control also meant to be composable? (I assume yes)

Reply:
Yep, it's the breadcrumbs composable

Reply:
Or i mean "will be" 😆